Gems/RecastNavigation/Code/Source/RecastNavigationSystemComponent.cpp

```diff
@@ -0,0 +1,83 @@
+
+#include <RecastNavigationSystemComponent.h>
+
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzCore/Serialization/EditContext.h>
+#include <AzCore/Serialization/EditContextConstants.inl>
+
+namespace RecastNavigation
+{
+    void RecastNavigationSystemComponent::Reflect(AZ::ReflectContext* context)
+    {
+        if (AZ::SerializeContext* serialize = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serialize->Class<RecastNavigationSystemComponent, AZ::Component>()
+                ->Version(0)
+                ;
+
+            if (AZ::EditContext* ec = serialize->GetEditContext())
+            {
+                ec->Class<RecastNavigationSystemComponent>("RecastNavigation", "[Description of functionality provided by this System Component]")
```

Comment:
Add a description as to what this component is

Reply:
Re-wrote the description. At the moment, this template component does nothing.